content/en/community/roadmap.md

```diff
@@ -0,0 +1,194 @@
+---
+title: OpenTelemetry Project Roadmap
+linkTitle: Roadmap
+aliases: [/roadmap]
+weight: 2
+---
+
+OpenTelemetry is a healthy open source community, and contributors are free to
+propose new workstreams or work on any part of the project that they desire.
+However, there’s also value in the project focusing community members of
+specific bodies of work and releases, as this allows us to form a more cohesive
+set of capabilities (unified semantics, single implementations for each
+language, etc.) and to ship the project in a more impactful way (new signal
+types release across multiple languages at once, etc.).
+
+This roadmap is not _law_, and it is not meant to be used to force people to
+work on specific projects - this is an open source project and we (community
+members, the governance committee, etc.) are not anyone’s boss. Rather, it
```

Comment:
@reyang can you raise a PR with the official version of the roadmap (https://github.com/open-telemetry/community/blob/main/roadmap.md) and make those changes there, this will be a copy of that document and we will try to keep it in sync. TY.

Reply:
Done via https://github.com/open-telemetry/community/pull/1361